integration-tests/utils/terra-sa-utils.js

```diff
@@ -0,0 +1,28 @@
+const _ = require('lodash/fp')
+const { JWT } = require('google-auth-library')
+const { bearerToken, userEmail, terraSaKeyJson } = require('../utils/integration-config')
+
+
+const makeAuthClient = _.memoize((subject, { client_email, private_key }) => {
```

Comment:
`_.memoize` uses the first argument to the function as a cache key. In this case, since we always pass the same subject, it will only do this once. It might be clearer to use `_.once` to make that explicit.

Reply:
That is true if we delete `getBearerToken.js`... and may remain true as this moves to a server context? I think `_.memoize` is the right thing as long as `getToken` has a `subject` parameter. If we find that we are always using a globally configured subject in all contexts, then I agree that this should be `_.once`. I'll keep an eye on that.